website/source/docs/internals/acl.html.markdown

```diff
@@ -161,6 +182,12 @@ and ACLs can be found [below](#discovery_acls).
 
 The policy for the "consul" service is always "write" as it is managed internally by Consul.
 
+User event policies are defined by coupling an event name prefix with a policy.
+The rules are enforced using a longest-prefix match policy. The default rule,
+applied to any user event without a matching policy, is provided by an empty
+string. An event policy is one of "read", "write", or "deny". Currently, only
+the "write" level is enforced during event firing. Events can always be read.
```

Comment:
Hmm hadn't thought of the watches. That's a fair question. The concern is mainly restricting access to the payload. I don't think just knowing "this event fired" is really an attack vector, but the payload can contain potentially sensitive information. I'll think more on this.